tests/unit/lms/services/h_api_test.py

```diff
@@ -132,27 +132,44 @@ def test_get_annotations(
 
         assert result == expected_result
 
-    def test_get_assignment_stats(self, h_api, http_service):
+    @pytest.mark.parametrize(
```

Comment:
This method are not very interesting, just make an HTTP request and return the response. Doing them both in one test.